android/app/src/main/java/com/zulipmobile/MainActivity.java

```diff
@@ -22,12 +20,4 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         WebView.setWebContentsDebuggingEnabled(true);
     }
-
-    @Override
-    public void onConfigurationChanged(Configuration newConfig) {
-        super.onConfigurationChanged(newConfig);
-        Intent intent = new Intent("onConfigurationChanged");
-        intent.putExtra("newConfig", newConfig);
-        this.sendBroadcast(intent);
-    }
```

Comment:
Interesting, good catch!

Let's remove this from `ReceiveShareActivity.kt` too. I puzzled a bit over that when reviewing #4124 which added it... and then saw it was copied from `MainActivity` and I guess I figured it was generic RN boilerplate, and didn't study it closer. (But in retrospect this story makes more sense -- if it were generic RN boilerplate they could just put it in the base class.)